data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -242,7 +241,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "Well, those things will certainly get their attention, and there’s nothing wrong with pitchforks. We easily outnumber the orcs, not to speak of the fact that they are killing each other."
+            message= _ "Well, those things will certainly get their attention, but there’s nothing wrong with pitchforks. We easily outnumber the orcs and they're even now killing each other."
```

Comment:
Can you make that apostrophe the unicode version instead?

Reply:
https://wiki.wesnoth.org/Typography_Style_Guide#Apostrophes

Reply:
(there’s one in "there’s" earlier in the string that you can copy from, btw)